modules/stochastic_tools/src/vectorpostprocessors/PolyChaosStatistics.C

```diff
@@ -0,0 +1,83 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// Stocastic Tools Includes
+#include "PolyChaosStatistics.h"
+
+registerMooseObject("StochasticToolsApp", PolyChaosStatistics);
+
+defineLegacyParams(PolyChaosStatistics);
+
+InputParameters
+PolyChaosStatistics::validParams()
+{
+  InputParameters params = GeneralVectorPostprocessor::validParams();
+  params.addClassDescription("Tool for calculating statistics with polynomial chaos expansion.");
+
+  params.addRequiredParam<UserObjectName>("pc_name", "Name of PolynomialChaos.");
+
+  MultiMooseEnum stats("mean=1 stddev=2 skewness=3 kurtosis=4");
+  params.addRequiredParam<MultiMooseEnum>("compute", stats, "Type of statistic to output.");
+  return params;
+}
+
+PolyChaosStatistics::PolyChaosStatistics(const InputParameters & parameters)
+  : GeneralVectorPostprocessor(parameters),
+    _pc_uo(getUserObject<PolynomialChaos>("pc_name")),
+    _type(getParam<MultiMooseEnum>("compute")),
+    _stat_type(declareVector("type")),
+    _stats(declareVector("value"))
+{
+  for (const auto & item : _type)
+    _stat_type.push_back(item.id());
+
+  _stats.resize(_stat_type.size());
+}
+
+void
+PolyChaosStatistics::execute()
+{
+  // Compute standard deviation
+  Real sig = 0.0;
+  if (_type.contains("stddev") || _type.contains("skewness") || _type.contains("kurtosis"))
+    sig = _pc_uo.computeSTD();
+
+  for (unsigned int t = 0; t < _type.size(); ++t)
```

Comment:
The MultiMooseEnum class has begin()/end() iterators. They return a MooseEnumItem which will act as a string and you can use the names instead of integers for clarity.